packages/backend/discovery/zksync2/ethereum/diffHistory.md

```diff
@@ -1,3 +1,275 @@
+-        "0xfe26699e"
++        "0x39607382"
+      values.facets.1.1.18:
+-        "0xef3f0bae"
++        "0xfe26699e"
+      values.facets.1.1.17:
+-        "0xb8c2f66f"
++        "0xef3f0bae"
+      values.facets.1.1.16:
+-        "0xdb1f0bf9"
++        "0xb8c2f66f"
+      values.facets.1.1.15:
+-        "0x33ce93fe"
++        "0xdb1f0bf9"
+      values.facets.1.1.14:
+-        "0x0ec6b0b7"
++        "0x33ce93fe"
+      values.facets.1.1.13:
+-        "0x631f4bac"
++        "0x0ec6b0b7"
+      values.facets.1.1.12:
+-        "0x8665b150"
++        "0x631f4bac"
+      values.facets.1.1.11:
+-        "0x7b30c8da"
++        "0x8665b150"
+      values.facets.1.1.10:
+-        "0x9d1b5a81"
++        "0x7b30c8da"
+      values.facets.1.1.9:
+-        "0xe5355c75"
++        "0x9d1b5a81"
+      values.facets.1.1.8:
+-        "0xfd791f3c"
++        "0xe5355c75"
+      values.facets.1.1.7:
+-        "0xd86970d8"
++        "0xfd791f3c"
+      values.facets.1.1.6:
+-        "0x4fc07d75"
++        "0xd86970d8"
+      values.facets.1.1.5:
+-        "0x79823c9a"
++        "0x4fc07d75"
+      values.facets.1.1.4:
+-        "0xa7cd63b7"
++        "0x79823c9a"
+      values.facets.1.0:
+-        "0xF3ACF6a03ea4a914B78Ec788624B25ceC37c14A4"
++        "0x5edb1756c0A0F933EB87f9d69DfA1db3167547a7"
+      values.facets.0.0:
+-        "0x409560DE546e057ce5bD5dB487EdF2bB5E785baB"
++        "0xAeA49FCEbe3A93ADaE67FF668C0ac87799537967"
+      values.getAllowList:
+-        "0x0C0dC1171258694635AA50cec5845aC1031cA6d7"
+      values.getGovernor:
+-        "0x4e4943346848c4867F81dFb37c4cA9C5715A7828"
++        "0x0b622A2061EaccAE1c664eBC3E868b8438e03F61"
+      values.getL2BootloaderBytecodeHash:
+-        "0x01000983d4ac4f797cf5c077e022f72284969b13248c2a8e9846f574bdeb5b88"
++        "0x01000831ba7021800f5d9103772fcc7463ed7e764a2a3624cacca6b3826172d0"
+      values.getL2DefaultAccountBytecodeHash:
+-        "0x01000651c5ae96f2aab07d720439e42491bb44c6384015e3a08e32620a4d582d"
++        "0x0100055bf7f1bc4237c2be24252fb6737cc235194139e544933c1dbf25c24ee8"
+      values.getProtocolVersion:
+-        18
++        19
+    }
+```
+
+```diff
+    contract L1ERC20Bridge (0x57891966931Eb4Bb6FB81430E6cE0A03AAbDe063) {
+      upgradeability.implementation:
+-        "0x03F3F3c12e11C2FAA60080bd3F7f80AADF369a33"
++        "0x79Cc1DF74Ac2d1B0876498C9FcE32c7e34F57B43"
+      upgradeability.admin:
+-        "0x4e4943346848c4867F81dFb37c4cA9C5715A7828"
++        "0x0b622A2061EaccAE1c664eBC3E868b8438e03F61"
+      implementations.0:
+-        "0x03F3F3c12e11C2FAA60080bd3F7f80AADF369a33"
++        "0x79Cc1DF74Ac2d1B0876498C9FcE32c7e34F57B43"
+    }
+```
+
+```diff
+    contract ValidatorTimelock (0xa0425d71cB1D6fb80E65a5361a04096E0672De03) {
+      values.owner:
+-        "0x4e4943346848c4867F81dFb37c4cA9C5715A7828"
++        "0x0b622A2061EaccAE1c664eBC3E868b8438e03F61"
+    }
+```
+
+```diff
++   Status: CREATED
+    contract Governance (0x0b622A2061EaccAE1c664eBC3E868b8438e03F61) {
```

Comment:
This contract needs to be added to the contracts section with the explanation provided here